### ISSUE TITLE
How do you have training techniques?

### ISSUE THREAD

**Issue by PuriHa** [hamuchiwa/AutoRCCar#168]
How do you have training techniques? 
I should press forward, left, right and hold or press it several times?
When I want the rc car to turn right, How should I press?
I try to do everything possible. But my car still doesn't turn correctly!
![S__22659076](https://user-images.githubusercontent.com/48644160/54494348-c334dc00-490b-11e9-9943-729d9074ee4f.jpg)
![S__22659077](https://user-images.githubusercontent.com/48644160/54494350-c334dc00-490b-11e9-92da-4e75fdbe677b.jpg)

Please tell me, I love this project very much, and I am very thoughtful. Can you answer my question? Thank you very much.

**Comment by hamuchiwa** [comment]
@PuriHa Besides the actual model tuning, keep your car centered, make sure images are mostly labeled correctly and your data is mostly balanced. If there are too many "forward" labels, try to drive on corner sections more. In the case the car is not centered, collecting some error correction data can also help.

You can try to tune the model in OpenCV as well, but I think what's better and simpler now is to create a CNN model in Tensorflow/Keras.

**Comment by PuriHa** [comment]
If I use Tensorflow, what steps should I do? How to do? 
Please help explain to me thoroughly. And then Code?
Thanks very much for your advice @hamuchiwa

**Comment by VishalWrench** [comment]
https://www.youtube.com/playlist?list=PLQVvvaa0QuDfhTox0AjmQ6tvTgMBZBEXN
may be this deep learning tutorial (CNN) using tensor-flow and Keras by Sentdex will help you. You can find many resources on internet about CNN. There is a whole series of self driving car (python plays gta 5 ) on his channel check it out.

**Comment by PuriHa** [comment]
And then file code **model.py** ? What can do? In what step?
